kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/base/CustomResourceDefinitionContext.java

```diff
@@ -72,7 +71,7 @@ public static CustomResourceDefinitionBuilder v1beta1CRDFromCustomResourceType(C
         .withNewSpec()
         .withGroup(instance.getGroup())
         .withVersion(version) // also set version to the first (and only) versions item
-        .addNewVersion().withName(version).withServed(true).withStorage(true).endVersion()
+        .addNewVersion().withName(version).withServed(instance.isServed()).withStorage(instance.isStorage()).endVersion()
```

Comment:
This change should also be applied to line 99

Reply:
Not possible as CustomResourceDefinitionBuilder is generated.

Reply:
![image](https://user-images.githubusercontent.com/488391/107251555-c1f78580-6a34-11eb-98b6-66ef030c1fd1.png)

Reply:
I don't follow why it shouldn't be possible.

Reply:
Ah sorry I misunderstood

Reply:
Done.